aws-observability/templates/sumologic_observability.master.template.yaml

```diff
@@ -461,7 +461,7 @@ Mappings:
   CommonData:
     NestedTemplate:
       BucketName: "sumologic-appdev-aws-sam-apps"
-      Version: "v2.10.0"
+      Version: "v2.9.0"
```

Comment:
I would recommend to also move all child templates and permission check templates to v2.9.0